Sources/SwiftCBOR/CBOREncoder.swift

```diff
@@ -1,3 +1,4 @@
+import Foundation
```

Comment:
eh, I guess I don't have a reason to avoid Foundation — that decision was made before the cross-platform (Linux) version of Swift was released, I thought it might not include Foundation, but it does.

Reply:
I've made the import conditional using `#if canImport(Foundation)`

Reply:
cool